ensembl_dbcopy/api/urls.py

```diff
@@ -49,8 +50,8 @@
 
 urlpatterns = [
     url(r'^', include(router.urls)),
-    url(r'^databases', ListDatabases.as_view(),name='databaselist'),
-    url(r'^tables$', ListTables.as_view(),name='tablelist'),
+    url(r'^databases/(?P<host>[\w-]+)/(?P<port>\d+)', ListDatabases.as_view(), name='databaselist'),
```

Comment:
Not directly related, but I would force `ListDatabases` and `ListTables` to be authenticated requests (except if they are used directly from client?) 
```
permission_classes = [IsAuthenticated, ]
```

Reply:
That's a good point, (and a new feature) I'd have to look into it further as these endpoints serve ajax requests from the frontend so I have to find a way to send a valid session token from JS.

Reply:
If those API calls are only made by Django backend, the authentication is already there. If it's  more complicated than that. forget it for now.

Reply:
Those API are called exclusively with jQuery ajax requests from the frontend.

Reply:
Forget about it for now then. We'll deal with that another day!